Add dump commands. Closes #102

SChinchi opened the pull request:
Add the following commands:

- `dump_buffs`
- `dump_inventories`
- `dump_stats {body}`
- `dump_state [target:<self>]`

![dump_commands](https://github.com/harbingerofme/DebugToolkit/assets/57867641/cc505004-4f21-474a-84ed-672227183f65)

The buff/inventory ones are useful for keeping track of who has what. The latter is also helpful to verify which characters get any NoTier items.

The stats one is mostly for wiki work, while the state one to get current stats during gameplay and/or skill cooldowns, entity state and AI values.

I wonder whether a new class should have been created instead to group all the commands, similar to `Lists.cs`, but that's all.

harbingerofme commented:
Seems mostly right, but make the logs go `Log.MessageNetworked(<blah>, args, LogLevel.MessageClientOnly);` to ensure they end up at the right person

SChinchi commented:
Ironically I had them initially as MessageNetworked but changed them because the commands aren't executed on the server.

Changed them as requested and also changed the related `Lang.COMMANDNAME` to `Lang.COMMANDNAME_HELP` to follow the naming convention properly.
